package.json

```diff
@@ -2,31 +2,32 @@
   "name": "yalesites-upstream",
   "license": "MIT",
   "scripts": {
-    "confim": "./scripts/local/confim.sh",
-    "confex": "./scripts/local/confex.sh",
-    "import-local-db": "./scripts/local/import-data.sh",
+    "assets-pull": "./scripts/local/assets-pull.sh",
     "build": "./scripts/local/build.sh",
-    "rebuild": "./scripts/local/rebuild.sh",
-    "export-db": "./scripts/local/export-db.sh",
-    "get-db": "./scripts/local/get-db.sh",
-    "get-files": "./scripts/local/get-files.sh",
-    "get-assets": "./scripts/local/get-files.sh",
-    "theme": "./scripts/local/theme.sh",
-    "build-theme": "./scripts/local/build-theme.sh",
-    "setup": "./scripts/local/setup.sh",
-    "local:theme-setup": "./scripts/local/dev-component-library-install.sh",
-    "prettier": "prettier '{.github,docs,web/profiles/custom,web/modules/custom,web/themes/contrib/atomic,web/themes/custom}/**/*.{json,md,js,html,scss,html}' --ignore-unknown --list-different",
-    "lint:styles": "stylelint --color {web/modules/custom,web/themes/contrib/atomic,web/themes/custom,web/profiles/custom}/**/*.scss",
+    "build-with-assets": "./scripts/local/build-with-assets.sh",
+    "confex": "./scripts/local/confex.sh",
+    "confim": "./scripts/local/confim.sh",
+    "db:create": "./scripts/local/db-create.sh",
+    "db:get": "./scripts/local/db-get.sh",
+    "db:import": "./scripts/local/db-import.sh",
+    "db:local-export": "./scripts/local/db-local-export.sh",
+    "files:get": "./scripts/local/files-get.sh",
+    "husky:commit-msg": "commitlint --edit $1",
+    "husky:pre-commit": "lint-staged",
+    "lint": "npm run lint:js && npm run lint:php && npm run lint:styles", 
     "lint:js": "eslint --color --no-error-on-unmatched-pattern --format stylish {web/modules/custom,web/themes/custom,web/profiles/custom}/**/*.js",
-    "local:cl-update": "./scripts/local/dev-component-library-update.sh",
     "lint:php": "composer lint:php",
-    "lint": "./scripts/local/lint.sh",
-    "test": "./scripts/local/test.sh",
-    "husky:commit-msg": "commitlint --edit $1",
-    "husky:pre-commit": "lint-staged"
+    "lint:styles": "stylelint --color {web/modules/custom,web/themes/contrib/atomic,web/themes/custom,web/profiles/custom}/**/*.scss",
+    "local:review-with-cl-branch": "./scripts/local/local-review-with-cl-branch.sh",
+    "local:review-with-atomic-branch": "./scripts/local/local-review-with-atomic-branch.sh",
+    "local:cl-dev": "chokidar 'web/themes/contrib/atomic/component-library-twig/components' -c 'npm run local:review-with-cl-branch'",
```

Comment:
I'm not able to figure out this line. I think you are already doing it, but we should describe some of these in a README file.